docs/samples/browser-plugin-meetings/app.js

```diff
@@ -658,9 +658,7 @@ function addPlayIfPausedEvents(mediaElements) {
 }
 
 function getCurrentMeeting() {
-  const meetings = webex.meetings.getAllMeetings();
-
-  return meetings[Object.keys(meetings)[0]];
+  return webex?.meetings?.getAllMeetings()[selectedMeetingId];
```

Comment:
this will won't work correctly if user selects a different meeting after they've joined one, but it's much better than what was before so worth merging anyway and we could raise a jira for the issue of clicking on other meetings after joining one